pkg/gather/clusterconfig/config_maps_test.go

```diff
@@ -58,6 +58,10 @@ func TestConfigMapAnonymizer(t *testing.T) {
 			mustNotFail(t, err, "error creating test data %+v")
 			cm := findMap(cml, tt.configMapName)
 			mustNotFail(t, cm != nil, "haven't found a ConfigMap %+v")
+			// just to make lint happy
+			if cm != nil {
+				return
+			}
```

Comment:
I don't think this is correct. I think we want to continue in the test when `cm != nil`.

Reply:
Oh, wait! Wrong operator. :facepalm: